src/ZkFold/Base/Protocol/ARK/Plonk.hs

```diff
@@ -58,9 +57,10 @@ data PlonkSetupParams = PlonkSetupParams {
         omega :: F,
         k1    :: F,
         k2    :: F,
-        gs    :: V.Vector G1,
```

Comment:
We need `gs` in the setup for the prover, though we do not need this field for the verifier. Consider Splitting the type for better `verify` efficiency.